lib/google/cloud/gemserver/cli/request.rb

```diff
@@ -45,7 +45,8 @@ class Request
           # gemserver was deployed to. Optional.
           def initialize url = nil, proj_name = nil
             gemserver_url = url.nil? == true ? remote(proj_name) : url
-            @http = Net::HTTP.new gemserver_url
+            port = Configuration.new.metadata[:platform] == "gke"? 8080 : 80
```

Comment:
Oddly enough, exposing on port 80 with GKE made the gem server unaccessible. Some additional settings might be necessary in `deployment.yaml.base`.